apis/privilegedaccessmanager/v1beta1/privilegedaccessmanagerentitlement_types.go

```diff
@@ -108,7 +108,7 @@ type PrivilegedAccessManagerEntitlementStatus struct {
 
 // PrivilegedAccessManagerEntitlementSpec defines the desired state of
 // PrivilegedAccessManagerEntitlement.
-// +kcc:proto=google.privilegedaccessmanager.v1.Entitlement
+// +kcc:proto=google.cloud.privilegedaccessmanager.v1.Entitlement
```

Comment:
I'm curious: How do we decide the path here?

Reply:
This essentially is the fully qualified name of the proto message, which the code generation tool should automatically populate.

https://github.com/justinsb/k8s-config-connector/blob/47ef09edfcb95e17b2fb8eced687219ffc0d3d50/dev/tools/controllerbuilder/template/apis/types.go#L75